packages/fast-components-react-msft/src/subheading/README.md

```diff
@@ -0,0 +1,8 @@
+# Subheading
+Use various subheading sizes to create a clear hierarchy within your page. Subheadings are paired with the Heading component.

```

Comment:
Please _italicize_ and lowercase component names.

Reply:
done, we might want to go through other component readmes because they don't all seem to be doing this